src/schema/v2/filterArtworksConnection.ts

```diff
@@ -230,7 +232,18 @@ export const filterArtworksArgs: GraphQLFieldConfigArgumentMap = {
   },
 }
 
-export const pageableFilterArtworksArgs = pageable(filterArtworksArgs)
+const pageableFilterArtworksArgs = pageable(filterArtworksArgs)
+
+const FilterArtworksInputType = new GraphQLInputObjectType({
```

Comment:
The new structured input type consists of all the root fields, plus the connection-paging ones.